frontend/ui/elements/screensaver_menu.lua

```diff
@@ -26,114 +26,144 @@ local function genMenuItem(text, setting, value, enabled_func, separator)
         separator = separator,
     }
 end
-
 return {
-    genMenuItem(_("Use last book's cover as screensaver"), "screensaver_type", "cover", hasLastFile),
-    genMenuItem(_("Use book status as screensaver"), "screensaver_type", "bookstatus", hasLastFile),
-    genMenuItem(_("Use random image from folder as screensaver"), "screensaver_type", "random_image"),
-    genMenuItem(_("Use document cover as screensaver"), "screensaver_type", "document_cover"),
-    genMenuItem(_("Use image as screensaver"), "screensaver_type", "image_file"),
-    genMenuItem(_("Use reading progress as screensaver"), "screensaver_type", "readingprogress", isReaderProgressEnabled),
-    genMenuItem(_("Leave screen as-is"), "screensaver_type", "disable", nil, true),
-    -- separator
-    {
-        text = _("Add message to screensaver"),
-        checked_func = function()
-            return G_reader_settings:isTrue("screensaver_show_message")
-        end,
-        callback = function()
-            G_reader_settings:toggle("screensaver_show_message")
-        end,
-        separator = true,
-    },
-    -- separator
     {
-        text = _("Settings"),
+        text = _("Screensaver"),
```

Comment:
Still a remnant Scredsaviour.

Reply:
yes, that is what we are calling the image, not the whole screen. so that the menu goes: `screen > sleep screen > screensaver`. Note `sleep screen` encompasses both `screensaver` and `screen message`.

Reply:
could be changed to `wallpaper` ...?